### PR TITLE
Clear QT_STYLE_OVERRIDE if the style was not found.

### PR DESCRIPTION
**Pull request by oskirby**
## Description
Some distros may set `QT_STYLE_OVERRIDE` by default when attempting to apply a uniform style their Qt5 applications. However, this style may not necessarily exist for Qt6, which will result in a failure to load QML content. To work around this issue, check that the style exists before creating the `QApplication`, and unset the environment variable if no such style was found.

The QML fails to load `QtQuick.Controls` when this situation occurs.

## Reference
Github: #3367
JIRA: [VPN-2113](https://mozilla-hub.atlassian.net/browse/VPN-2113)

## Checklist
    
- [x] My code follows the style guidelines for this project
- [x] I have not added any packages that contain high risk or unknown licenses (GPL,  LGPL, MPL, etc. consult with DevOps if in question)
- [x] I have performed a self review of my own code
- [x] I have commented my code PARTICULARLY in hard to understand areas
- [x] I have added thorough tests where needed

### CODE REVIEW

File: linux/debian/control.qt6 [hunk 1]
```diff
@@ -17,6 +17,8 @@ Build-Depends: debhelper (>= 9.20160709),
                libqt6opengl6-dev (>=6.2.0~),
                libqt6svg6-dev (>=6.2.0~),
                libqt6websockets6-dev (>=6.2.0~),
+               libssl-dev,
```
Comment (bakulf) [inline linux/debian/control.qt6:20]: Unrelated change

File: src/command.cpp [hunk 2]
```diff
@@ -181,6 +182,12 @@ int Command::runQmlApp(std::function<int()>&& a_callback) {
 
   qInstallMessageHandler(LogHandler::messageQTHandler);
 
+  QString styleName = qEnvironmentVariable("QT_STYLE_OVERRIDE");
```
Comment (bakulf) [inline src/command.cpp:185]: Can we simply unset it always? Maybe on Linux only

File: linux/debian/control.qt6 (hunk 1 above)
Comment (oskirby) [reply linux/debian/control.qt6:20]: I can move this into a separate PR

**Review by bakulf — APPROVED** [review]
